### ISSUE TITLE
Resource Report - 2.4.6 Headings and Labels - 1 skipped heading level is present on the report title

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9882]
As a user reliant on a screen reader, I expect heading levels to be in a sequential order so that I can form a mental image of the structure of the page and be able to skip to content quickly.

**Linked Work Item:**
#9663  

**Expected Result:**
Heading levels should be ordered sequentially so that screen reader/keyboard users can build a mental image of the structure of the page and skip to content easily without worrying whether they have missed important information.

**Actual Result:**
The report title is at H4, meaning that there is no H3 heading and users may believe they have missed out on important info.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/b50628bc-57b7-4cfa-91b1-b9552766ea27)

**Comment by johnatawnclementawn** [comment]
Resolved in #10063

**Comment by imogenroberts** [comment]
RETEST: Skipped heading level has now been resolved - the affected heading is reassigned to H4.

**Evidence:**
![image](https://github.com/archesproject/arches/assets/77161085/adc7c89d-eccf-4eef-ba02-8376b5a28304)
